src/client/testing/utils/date-utility.ts

```diff
@@ -17,3 +19,32 @@ export const getDateInFuture = (
     year: futureDate.format('YYYY'),
   };
 };
+
+export const geRequiredtDateInFormat = (
+  numberOfDaysFromToday: number,
+  requiredformat: string,
+) => {
+  const date = new Date();
+  let requiredDate = '';
+  const futureDate = date.setDate(date.getDate() + numberOfDaysFromToday);
+  requiredDate = dayjs(futureDate).format(requiredformat);
+  return requiredDate;
+};
+
+export const getWeekRange = (numberOfDaysFromToday: number) => {
+  const date = dayjs().add(numberOfDaysFromToday, 'day').format('YYYY-MM-DD');
+  const startOfWeek = dayjs(date).startOf('isoWeek');
+  const endOfWeek = dayjs(date).endOf('isoWeek');
+  const formattedStartOfWeek = startOfWeek.format('D MMMM');
+  const formattedendOfWeek = endOfWeek.format('D MMMM');
+  return `${formattedStartOfWeek} to ${formattedendOfWeek}`;
+};
+
+export const getWeekRangeForRequiredDate = (requiredDate: string) => {
+  const date = dayjs(requiredDate).format('YYYY-MM-DD');
+  const startOfWeek = dayjs(date).startOf('isoWeek');
+  const endOfWeek = dayjs(date).endOf('isoWeek');
+  const formattedStartOfWeek = startOfWeek.format('D MMMM');
+  const formattedendOfWeek = endOfWeek.format('D MMMM');
+  return `${formattedStartOfWeek} to ${formattedendOfWeek}`;
+};
```

Comment:
These methods are identical apart from their first line, so we could refactor them to avoid this duplication, e.g. 
<img width="643" alt="image" src="https://github.com/user-attachments/assets/d71d72e0-3264-46ad-965e-0ea1430bb1fe" />